tools/cloud-build/daily-tests/ansible_playbooks/test-mounts-and-partitions.yml

```diff
@@ -13,7 +13,16 @@
 # limitations under the License.
 
 ---
+- name: Assert variables are defined
+  ansible.builtin.assert:
+    that:
+    - custom_vars.partitions is defined
+    - custom_vars.mounts is defined
 
+- name: Include test mounts
```

Comment:
Sounds like `test_mounts` should be a separate test then. This could be converted to `test_partitions` and added as a test alongside `test_mounts`. I imagine it would still take `mounts` as a parameter to test whether the mounts exist on the partitions.

Reply:
Ya, I debated this. It is tricky because the original test is not only testing mounts on the login node but also testing them on each partition. We probably still want testing mounts on each partition as part the partition test but it should not be in the mounts only test. 

How it was structured:
- test-mounts-and-partitions.yml
  - tests mounts
  - tests partitions
  - tests mounts on partitions

How the pr is currently written:
- test-mounts.yml
  - tests mounts
- test-mounts-and-partitions.yml
  - tests mounts (via test-mounts.yml)
  - tests partitions
  - tests mounts on partitions

Alternative option:
- test-mounts.yml
  - tests mounts
- test-partitions.yml (note: is this naming reasonable)
  - tests partitions
  - tests mounts on partitions

Is your preference the alternative option? Obviously then we would add both as post deploy tests that previously had test-mounts-and-partitions.yml.

Reply:
> Is your preference the alternative option? Obviously then we would add both as post deploy tests that previously had test-mounts-and-partitions.yml.

Yes, that's how I'd organize it. I'm also open to creating follow-on task to do that since tests are already running.

Reply:
Done.